docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,140 @@
+---
+meta:
+  title: "Webhooks"
+  parentTitle: "Guides"
+  description: "Learn Liveblocks Webhooks concepts and implementation"
+---
+
+## Webhooks
+
+In this guide, we will simplify how to use webhooks within your product, to ensure that you are able to store and process Liveblocks data real time.  Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.
```

Comment:
```suggestion
In this guide, we will explain how to use webhooks within your product, to ensure that you are able to store and process Liveblocks data real time.  Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.
```

Are we simplifying or explaining? That would be the only place where we talk about it, isn't it?

Reply:
Explaining makes more sense here. I think i jumped between the two when editing but I agree with this 👍